gravity-forms/gw-list-field-as-choices.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+* Gravity Wiz // Gravity Forms // Use List Field as Choices for Gravity Forms
+* https://gravitywiz.com/use-list-field-choices-gravity-forms/
+*
+* Adds support for populating choice-based fields (i.e. checkboxes, selects, radio buttons) with values entered in a 
+* List field. This functionality requires that the form has multiple pages and that the List field must be placed on 
+* a page prior to the choice-based field that it will populate.
+*
+* Plugin Name:  Gravity Forms - Use List Field as Choices for Gravity Forms
```

Comment:
Let's drop the second "Gravity Forms" from the plugin name. Should look like this:
> Gravity Forms - Use List Field as Choices